src/components/shared/Button/Button.scss

```diff
@@ -0,0 +1,36 @@
+.button {
+  letter-spacing: 0.1rem;
+  cursor: pointer;
+  font-size: 0.8rem;
+  line-height: 2.5rem;
+  position: relative;
+  text-decoration: none;
+  text-transform: uppercase;
+  width: 40%;
+  height: 5rem;
+
+  &__span {
+    font-size: 1.2rem;
+  }
+}
+
+.effect {
```

Comment:
i to do poprawy trzeba rozkminic bardziej dynamiczna wartosc w ktorej bedzie mozna w propsach nadac kolor ? do przemyslenia kwestia